Documentation/API Backend/openrpc/components/schemas/state.json

```diff
@@ -0,0 +1,11 @@
+{
+    "type": "object",
+    "properties": {
+        "robotOnline": {
+            "type": "boolean"
```

Comment:
we can define an enum

Reply:
We have to be careful about this. Most operators manuals for industrial robots have a state chapter (including error states) that are dozens of pages long. This could be daunting to write out on an enum

Reply:
the robotOnline was literally just put in as an example value. The point of this was just to establish a format that people like you can use to develop the details

Reply:
Good point on the enum getting pretty big - I can see us maybe having a hierarchy of states, eg the main state is error, the substate is error-hardware-overcurrent and there's also an associated error message.

Reply:
On second thoughts maybe the state is just presented as a string (for showing to the user) and anything that needs to make decisions checks state flags which the controller toggles based on the state, eg `jog_enabled`, `has_error` - that keeps the state logic more contained inside the controller